CZI_mpl.tex

```diff
@@ -65,45 +64,32 @@ \subsection{Fostering downstream projects}
 
-To maintain Matplotlib's health we need to:
+We have accumulated about 1200 open issues and 300 open PRs due to this imbalance. There may be critical bug reports or insightful feature requests among the former, while among the latter are useful contributions or bug fixes that would improve Matplotlib for direct users and downstream packages. The backlog is discouraging for new and occasional contributor and distracting for core developers.
 
+To maintain Matplotlib's health we need to:
+%% these can probably be shortened further, to just stuff like Triage and on board 
+%%since explainations are in text
 \begin{itemize}[noitemsep]
-\item Review and label current backlog of Issues and PRs in terms of
-  topic, difficulty, and urgency.  Promptly review and label newly
-  opened Issues and PRs
-\item Fix critical bugs and regressions promptly.
-\item On-board new contributors to the project team; this is critical to
-  sustaining and diversifying our developer community.
-  % AL had concerns about this being an agressive growth target, I think
-  % of this more in terms of making sure we are bring people on to offset
-  % the people eventually rolling off (for what ever reason)
-  % JMK: We hav a backlog of PRs largely because we don't have enough
-  % people to review them. I think a larger community is key.
-\item Maintain backward compatibility.  If we break the API, ensure it
-  is intentional and well documented.
-\item Manage discussions about proposed enhancements, features, and API changes.
+\item Fix critical bugs and regressions
+\item Triage backlog of issues and PRs in terms of
+  topic, difficulty, and urgency and promptly triage newly
+  opened issues and PRs
+\item Maintain backward compatibility and document extensively for intentional changes
+\item On-board new contributors to sustain and diversify developer team
+\item Manage discussions about proposed enhancements, features, and API changes
 \end{itemize}
-The requested support for developers is intended to complement and
-facilitate, not replace, the crucial work by volunteers.  We aim to
-better co-ordinate and nurture their
-efforts, with the goal of growing and sustaining a diverse community
-of expert contributors, both volunteer and paid.  We view
-this proposal as the start of a new phase for Matplotlib, in which we
-will need to find support from a variety of sources in the coming years.
+
+The requested support for developers is intended to complement and facilitate, not replace, crucial volunteer work. We aim to better co-ordinate and nurture their efforts, with the goal of growing and sustaining a diverse community of volunteer and paid expert contributors. 
+
+%% We view this proposal as the start of a new phase for Matplotlib, in which we will need to find support from a variety of sources in the coming years.
+%% - not sure why this sentence needs to be here
```

Comment:
The point is that we recognize the CZI funding is not a one-shot solution--we will need paid developers indefinitely, so we will need to find additional sources of support in the future.